sites/vehicleservicepros.com/server/styles/index.scss

```diff
@@ -23,3 +23,17 @@ $theme-site-header-breakpoints: (
 );
 
 @import "../../node_modules/@endeavor-business-media/package-shared/scss/cygnus-skin";
+
+.with-diagnostic-feed {
+  .directory-facets {
```

Comment:
Instead of selecting based on two different blocks, can you create a `.directory-facets` modifier these styles there instead? For example:

```scss
.directory-facets {
  $self: &;
  &--diagnotic-feed {
    #{ $self } {
      &__list {}
    }
  }
}
```